hometask1_1-java_14-features/src/main/java/ui/pages/LoginPage.java

```diff
@@ -0,0 +1,23 @@
+package ui.pages;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+public class LoginPage {
+
+    public SelenideElement getNameInput() {
+        return $("#user-name").should(Condition.visible);
+    }
+
+    public SelenideElement getPasswordInput() {
+
+        return $("#password").should(Condition.visible);
+    }
+
+    public SelenideElement getLoginButton() {
+
+        return $("#login-button").should(Condition.visible);
+    }
```

Comment:
Try to avoid hardcoding here please. Create private variables for all elements, and reuse them in all methods